lightning/src/routing/mod.rs

```diff
@@ -12,3 +12,4 @@
 pub mod gossip;
 pub mod router;
 pub mod scoring;
+pub mod pickhardt_router;
```

Comment:
I looked at the Rust handbook for defining modules, and there are some interesting things in it, for example the router module could be either router.rs or router/mod.rs but not both.

We can reexport modules (pub use ... instead of pub mod), but in my opinion it will make it much harder to navigate the code base.

One option could be not having an extra subdirectory and putting large_payment_router.rs and min_cost_lib.rs in the routing directory directly.